analyzers/src/SonarAnalyzer.VisualBasic/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -0,0 +1,35 @@
+﻿/*
```

Comment:
I kept the VB stub for now, but don't register it. Just drop me a note and I will delete it.

Reply:
Generally, it would be better to do it later. 
We're sure we'll use it. I don't mind having it ready as you've already put an effort to write it.
let's keep it as it already exists.

Reply:
Ah, the coverage bites :/ Let's remove it then (keep it somewhere so you can paste it when needed)